tests/TestController.py

```diff
@@ -21,13 +21,15 @@ def setUp(self):
             'name':'test event',
```

Comment:
Test files should be named as follows: test_controller.py instead of TestController.py
Basically, the name of any test file should begin with the word test, then an underscore and then followed by the name of the file that the tests are being written for.